### PR TITLE
[qa] Replace with openwisp-utils-qa-checks

### PR DESCRIPTION
**Pull request by waleko**
Replace old qa checks with `openwisp-utils-qa-checks` (related to openwisp/openwisp-utils#63 for this repo)

### CODE REVIEW

**Comment by waleko** [comment]
Travis is failing because of absence of endlines ([travis](https://travis-ci.org/openwisp/django-freeradius/jobs/631862059#L1125))

**Comment by atb00ker** [comment]
> Travis is failing because of absence of endlines (travis)

Again, needs to be fixed in this PR! :smile:

**Comment by waleko** [comment]
Travis is failing with `SAMPLE_APP=1`. Previously, there was a simple workaround:
https://github.com/openwisp/django-freeradius/blob/4fbf6ca98cea726d22dae54a0684c9468fc752a1/.travis.yml#L50
Perhaps, we can make an `if` to skip the check if `SAMPLE_APP` is 1.

**Comment by atb00ker** [comment]
> Perhaps, we can make an if to skip the check if SAMPLE_APP is 1.
That sounds good!

File: .travis.yml [hunk 1]
```diff
@@ -28,42 +28,29 @@ env:
...
-  # ensure no migrations pending but show output for easier debugging
+  - >
+    openwisp-utils-qa-checks --migration-path "./django_freeradius/migrations ./tests/sample_radius/migrations"
+    --skip-checkmakemigrations 
   - |
     if [[ $TRAVIS_PYTHON_VERSION == 3.5 && $SAMPLE_APP != "1" ]]; then
-      OUTPUT=$(./tests/manage.py makemigrations --dry-run django_freeradius)
-      echo $OUTPUT
-      echo $OUTPUT | grep "No changes detected"
+      openwisp-utils-qa-checks --skip-isort --skip-flake8 --skip-checkmigrations --skip-checkendline --skip-checkcommit --migration-module django_freeradius
```
Comment (nemesisdesign) [inline .travis.yml]: please use mult-iline statementsfor better readability

**Pull request merged**